resilience4j-spring-boot2/build.gradle

```diff
@@ -1,6 +1,9 @@
 dependencies {
     compile project(':resilience4j-spring')
     compile project(':resilience4j-micrometer')
+    compile project(':resilience4j-reactor')
+    compile(libraries.spring_boot2_webflux)
+    compileOnly(libraries.rxjava2)
```

Comment:
Is this needed?

Reply:
you are right. I forgot to remove this at the end. Will remove